src/Sylius/Bundle/OrderBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -56,6 +56,22 @@ public function findLatest($count)
     /**
      * {@inheritdoc}
      */
+    public function createCartsPaginator(array $criteria = [], array $sorting = [])
```

Comment:
It could be slightly harder, as we don't have a `cart` resource... So it would create order routing